### DISCUSSION TITLE
Conda Forge Support

### DISCUSSION THREAD

**Discussion by joshmedeski** [goreleaser/goreleaser#5390] [Ideas, Issue Triage, and General discussions]
It would be great to add Conda Forge support for GoReleaser. I have a community member that wants me to add support for this tooling.

https://conda-forge.org/docs/maintainer/adding_pkgs/#creating-recipes

**Comment by caarlos0** [comment]
Well, since no one else showed interest in this, I think I'm gonna leave this for the long future.

Will close the here, but if anyone in the future also thinks this should happen, please feel free to comment here :) 

BTW Thanks for the suggestions, Josh!

Cheers!
